bip-schnorr.mediawiki

```diff
@@ -120,10 +122,7 @@ All provided signatures are valid with overwhelming probability if and only if t
 ** Let ''r = int(sig<sub>i</sub>[0:32])''; fail if ''r &ge; p''.
 ** Let ''s<sub>i</sub> = int(sig<sub>i</sub>[32:64])''; fail if ''s<sub>i</sub> &ge; n''.
 ** Let ''e<sub>i</sub> = int(hash(bytes(r) || bytes(P<sub>i</sub>) || m<sub>i</sub>)) mod n''.
-** Let ''c = r<sup>3</sup> + 7 mod p''.
-** Let ''y = c<sup>(p+1)/4</sup> mod p''.
-** Fail if ''c &ne; y<sup>2</sup> mod p''.
-** Let ''R<sub>i</sub> = (r, y)''.
+** Let ''R<sub>i</sub> = lift_x(r)''; fail if ''lift_x(r)'' fails.
```

Comment:
I like `lift_x` conceptually, but the exact computations necessary seem to be missing now.

Reply:
Yes, that's true... I changed it that way because at the moment we do the same for the `point` function, which also can fail due to reasons only in the footnote (or inlined in this PR). 

Maybe it's easier to keep most of the other text as a footnote -- this seem to be in the spirit of the other footnotes -- and add a short sentence to `point` and `lift_x()` that explains how it can be implemented?

Reply:
I think better than a sentence would be an algorithmic description (i.e. the thing you replaced). Maybe a subroutine? But since we use lift_x only once that's maybe overkill.

Reply:
Let me try that.

Reply:
Looks better now